helm_deploy/whereabouts-api/templates/ingress-v1-2.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
Not sure whether this will work - can't really review it

Reply:
its just coped from the other ingress.  its same value